<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
    Copyright (C) 2004-2011 Martin Jolicoeur (snmpb1@gmail.com) 

    This file is part of the SnmpB project 
    (http://sourceforge.net/projects/snmpb)

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 2 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see http://www.gnu.org/licenses/.
</comment>
 <class>AgentProfile</class>
 <widget class="QDialog" name="AgentProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Agent Profiles</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/snmpb.png</normaloff>images/snmpb.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="OKCancelBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="ProfileTree">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="ProfileProps">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ProfileGeneral">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="1" column="0">
        <widget class="QGroupBox" name="Profile">
         <property name="title">
          <string>Profile</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="ProfileNameL">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ProfileName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="TargetAgent">
         <property name="title">
          <string>Target SNMP Agent</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="AddressL">
            <property name="text">
             <string>Address/hostname</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Address">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Port">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="PortL">
            <property name="text">
             <string>Agent Port</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="TimeoutRetries">
         <property name="title">
          <string>Timeout and Retries</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <property name="bottomMargin">
           <number>11</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="TimeoutL">
            <property name="text">
             <string>Timeout (sec)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="RetriesL">
            <property name="text">
             <string>Retries</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="Retries">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="Timeout">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="Version">
         <property name="toolTip">
          <string>Select the SNMP version to use</string>
         </property>
         <property name="title">
          <string>Supported SNMP Version</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="V2">
            <property name="text">
             <string notr="true">SNMPv2</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="V1">
            <property name="text">
             <string notr="true">SNMPv1</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="V3">
            <property name="text">
             <string notr="true">SNMPv3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="GeneralL">
         <property name="text">
          <string>&lt;span style=&quot;font-family:'Sans Serif'; font-size:18pt; font-weight:600;&quot;&gt;General Properties&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ProfileV1V2c">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="Communities">
         <property name="toolTip">
          <string>Community strings specific to SNMPv1/v2c can be entered here</string>
         </property>
         <property name="title">
          <string extracomment="This has nothing to do with online/offline groups of people; rather, this is SNMP specific term, designating a piece of text used roughly as a password (or more like, a hidden directory name).">Communities</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <property name="bottomMargin">
           <number>11</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ReadComm">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="ReadCommL">
            <property name="text">
             <string>Read community</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="WriteComm">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="WriteCommL">
            <property name="text">
             <string>Write community</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="V1V2cL">
         <property name="text">
          <string>&lt;span style=&quot;font-family:'Sans Serif'; font-size:18pt; font-weight:600;&quot;&gt;SNMPv1 / v2c Properties&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ProfileBulk">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="3" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="BulkL">
         <property name="text">
          <string>&lt;span style=&quot;font-family:'Sans Serif'; font-size:18pt; font-weight:600;&quot;&gt;Get-Bulk Properties&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="GetBulk">
         <property name="title">
          <string>Get-Bulk</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="1">
           <widget class="QSpinBox" name="MaxRepetitions">
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="NonRepeaters">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="NonRepeatersL">
            <property name="text">
             <string extracomment="this parameter is currently unused">Non repeaters</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="MaxRepetitionsL">
            <property name="text">
             <string extracomment="this parameter is currently unused">Max repetitions</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ProfileV3">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="4" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="V3Context">
         <property name="title">
          <string>SNMPv3 context</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ContextName"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ContextEngineID"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="ContextNameL">
            <property name="text">
             <string>Context Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ContextEngineIDL">
            <property name="text">
             <string>Context Engine ID</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="V3User">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>SNMPv3 user (USM)</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="SecLevelL">
            <property name="text">
             <string>Security Level</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="SecNameL">
            <property name="text">
             <string>Security Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="SecLevel">
            <item>
             <property name="text">
              <string extracomment="no authentication, no privacy">noAuthNoPriv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="authenticated plaintext (no privacy)">authNoPriv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string extracomment="both authentication and privacy protection (traffic encryption)">authPriv</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="SecName"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="V3L">
         <property name="text">
          <string>&lt;span style=&quot;font-family:'Sans Serif'; font-size:18pt; font-weight:600;&quot;&gt;SNMPv3 Properties&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ProfileTree</tabstop>
  <tabstop>ProfileName</tabstop>
  <tabstop>Address</tabstop>
  <tabstop>Port</tabstop>
  <tabstop>Retries</tabstop>
  <tabstop>Timeout</tabstop>
  <tabstop>V1</tabstop>
  <tabstop>V2</tabstop>
  <tabstop>V3</tabstop>
  <tabstop>ReadComm</tabstop>
  <tabstop>WriteComm</tabstop>
  <tabstop>NonRepeaters</tabstop>
  <tabstop>MaxRepetitions</tabstop>
  <tabstop>SecName</tabstop>
  <tabstop>SecLevel</tabstop>
  <tabstop>ContextName</tabstop>
  <tabstop>ContextEngineID</tabstop>
  <tabstop>OKCancelBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>OKCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>AgentProfile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OKCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>AgentProfile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
